client/finality-grandpa/src/lib.rs

```diff
@@ -52,6 +52,10 @@
 //! or prune any signaled changes based on whether the signaling block is
 //! included in the newly-finalized chain.
 
+#![warn(missing_docs)]
+
+pub use finality_grandpa::voter::report;
```

Comment:
There is a bunch of pub use in L122, I think this should be moved there as well.

Reply:
I was looking at the file and I noticed that things seems to be divided into two sections, before and after the `afg_log` macro

The ones on L122 seems to refer to entities in the current crate? while the ones at the top are related to other crates?

That's the rationale for me placing at the top, however I'm not sure here...